src/System.Management.Automation/engine/Interop/Windows/WNetAddConnection2.cs

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Windows
+    {
+        internal const int CONNECT_NOPERSIST = 0x00000000;
+        internal const int CONNECT_UPDATE_PROFILE = 0x00000001;
+        internal const int RESOURCE_GLOBALNET = 0x00000002;
+        internal const int RESOURCETYPE_ANY = 0x00000000;
+        internal const int RESOURCEDISPLAYTYPE_GENERIC = 0x00000000;
+        internal const int RESOURCEUSAGE_CONNECTABLE = 0x00000001;
+
+        [StructLayout(LayoutKind.Sequential)]
+        internal unsafe struct NetResource
```

Comment:
Should this reflect the unmanaged type `NETRESOURCEW`?

Reply:
Done.